.github/workflows/docker.yml

```diff
@@ -0,0 +1,39 @@
+name: docker
+
+on:
+  push:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  docker-build-push:
+    strategy:
+      matrix:
+        NODE_VERSION: [10.15.3, 10.17.0, 12, 14, 16]
```

Comment:
Not at all, this will use a newer version of Alpine (again, with a bunch of fixes we're currently missing). It might be safer to use version=10. This way we wont overwrite the existing tag. If anything breaks, we can go back to the old image then. WDYT?